### DISCUSSION TITLE
Importing faces

### DISCUSSION THREAD

**Discussion by alandefreitas** [photoprism/photoprism#2410] [Q&A] [answered]
Hi!

Does PhotoPrism import faces/people that already exist in the photo metadata? I'm trying to import my photos but this metadata seems to be ignored.

The faces in People seem to be new faces PhotoPrism recognize because they are in "New", there are much fewer faces than tagged in the originals, the faces have no names, and the same face in the originals is often split into more than one person.

I usually use Digikam to organize the photos and tag everyone, then PhotoPrism to explore them. The metadata is always embedded in the photo as exif and stored in the sidecar file.

**Comment by graciousgrey** [comment, marked as answer]
Faces tagged with digikam are not imported yet. But it's planned: https://github.com/photoprism/photoprism/issues/747 :)

**Reply by alandefreitas** [reply]
I had seen #747 before but I thought it had become a discussion about keywords only.

Thanks @graciousgrey !
